DoSomethingArchive/legacy-website
# writes script to batch exclude reportbacks

#### What's this PR do?

Provides a new custom drush command to batch exclude reportbacks to clear campaign inboxes. 
Command can be used for any campaign and campaign run by taking the following parameters: 
-`nid` - campaign nid 
-`run_nid` - campaign run nid 
-`quantity_limit` - threshold for exclusion - reportback quantity will be less than this number
#### How should this be manually tested?

run `drush batch-exclude-reportbacks` command with above parameters 
check campaign reportbacks and make sure only reportbacks that are under the quantity threshold remain in the reportback inbox

e.g. Run `drush batch-exclude-reportbacks 1321 1837 200`. On your local, visit http://dev.dosomething.org:8888/us/node/1321/rb and make sure only reportbacks with quantities over `200` remain in the inbox.
#### What are the relevant tickets?

Fixes #6392
